wisp-backoff/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+  kotlin("jvm")
+  `java-library`
+}
+
+dependencies {
+  api(Dependencies.wispConfig)
+  api(Dependencies.wispLease)
+  api(Dependencies.wispLeaseTesting)
+  api(Dependencies.wispLogging)
+  api(Dependencies.wispResourceLoader)
+  api(Dependencies.wispResourceLoaderTesting)
+  api(Dependencies.wispSsl)
+  implementation(Dependencies.bouncycastle)
+  implementation(Dependencies.guice)
```

Comment:
This needs to be removed for Wisp, and the other deps checked to reduce to the minimum required.